zookeeper/core/component_test.py

```diff
@@ -646,6 +647,64 @@ class Parent:
     assert instance.child.a == 5
 
 
+def test_component_configure_override():
+    @component
+    class A:
+        def __configure__(self, conf, **kwargs):
+            pass
+
+    # It doesn't call `configure`, so an error should be thrown.
+    with pytest.raises(
+        ValueError,
+        match=re.escape(
+            "`A` remains unconfigured after calling __configure__! Make sure to call"
+            " `configure(self, conf, **kwargs)` at the end of this function."
```

Comment:
What characters need to be escaped here? I would also just test the prefix of this message to be honest, no need to test for the entire thing (not that it really matters).

```suggestion
            "`A` remains unconfigured after calling __configure__!"
```

Reply:
The asterisks need to be escaped